styles.css

```diff
@@ -8,38 +8,101 @@ body {
   padding: 0;
   margin: 0;
   font-family: "Inter", sans-serif;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
 }
 
 /* The section below styles the header (logo and menu) */
 header #navigation {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
   position: fixed;
-  overflow: hidden;
   width: 100%;
-  height: 60px;
-  background-color: white;
+  padding: 20px 24px;
+  margin: 0;
+}
+
+header #navigation .nav-header {
   display: flex;
   justify-content: space-between;
   align-items: center;
-  padding: 12px 20px;
+  margin-right: 50px;
 }
 
-header #navigation #logo,
-header #navigation #menu {
-  font-size: 24px;
+header #navigation .nav-header #logo {
+  font-size: 20px;
   font-weight: 600;
   line-height: 28px;
-  text-decoration: none;
   color: #28352f;
 }
 
-header #navigation #menu {
-  margin-right: 60px;
+header #navigation .nav-header .nav-toggle {
+  font-size: 20px;
+  font-weight: 600;
+  line-height: 28px;
+  color: #28352f;
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
 }
 
-header #navigation #nav-list {
+header #navigation .nav-list {
+  height: 0;
+  background: #fff;
+  position: fixed;
+  top: -20px;
+  left: 0;
+  right: 0;
+  z-index: 10;
+  transition: all 1s ease-in-out;
   display: none;
 }
 
+footer .email-footer h3 a {
+  text-decoration: none;
+  font-size: 20px;
+  line-height: 30px;
+  font-weight: 400;
+  color: #42526e;
+}
+
+header #navigation .nav-list .nav-link a {
+  font-size: 28px;
+  line-height: 44px;
+  font-weight: 600;
+  color: #3a4a42;
+}
+
+header #navigation .nav-list .nav-close {
+  display: block;
+  position: absolute;
+  top: 40px;
+  right: 30px;
+  background-color: transparent;
+  border: none;
+  font-size: 28px;
+  font-weight: 600;
+  line-height: 28px;
+  color: #67798e;
+  cursor: pointer;
+}
+
+header #navigation .nav-list.active {
+  height: 50vh;
```

Comment:
- Kindly, your mobile menu has a larger space on the bottom, kindly fix it and stick the Figma design and have a better UI 👍👌


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/215751643-46bad15c-6707-4862-9595-1bb6bbf4472d.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/215751639-b3715fd1-52f0-49a4-bc3b-86a16118bb24.png" width="200">|


😎️